### DISCUSSION TITLE
Unable to apply to Octernship as there were no octernships opportunities listed out.

### DISCUSSION THREAD

**Discussion by auniraden** [community/community#66555] [GitHub Education] [answered]
I wanted to apply for an octernship, however, there are no octernships listed out. I would like to know when do the octernships opportunities available to apply?

**Comment by arafajs** [comment, marked as answer]
he GitHub Octernships program is currently in its off-season. The next application cycle will open in January 2024. You can check the GitHub Octernships website for updates on the application timeline.

In the meantime, you can still prepare for the application by:

- Becoming a verified student on GitHub Global Campus.
- Becoming an active contributor on GitHub.
- Enrolling in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia.
- Learning about open source software development.
- Practicing your coding skills.

**Reply by auniraden** [reply]
Oh thank you so much!

**Reply by shivang21007** [reply]
Thankyou @arafajs for helping us.

**Comment by ojuschugh1** [comment]
Hi @arafajs  Is the github octernship started?

**Reply by VatsShaurya** [reply]
Did you find anything? I got to know it is discontinued.
